widgets/lib/steppers/steppers_model.dart

```diff
@@ -0,0 +1,41 @@
+import 'package:flutter/material.dart';
+
+/// This style model is used to customize the stepper states that are `completed` `active` and `inactive`.
+/// You can change the styles for the items based on the states.
+class StepperStyle {
```

Comment:
The class is not used in the file, either consider removing it or moving it to separate file and import when needed.

Reply:
This is used in the woi_steppers.dart file by WOIStepper widget for 3 variables.

Reply:
Okay I was just looking and standard placements. but looks good for now.